scripts/lib/wic/canned-wks/mkefidisk-multiboot.wks

```diff
@@ -4,9 +4,9 @@
 
 part /boot --source bootimg-efi --sourceparams="loader=grub-efi" --ondisk sda --label msdos --active --align 1024
 
-part / --source rootfs --ondisk sda --fstype=ext4 --label platform --align 1024
+part / --source rootfs --ondisk sda --fstype=ext4 --label platform --size 2756 --align 1024
```

Comment:
Again would be better to use the overhead factor explicitly here. So the one knows why the particular size is chosen. 
Better would be to set the --size to the final size and set overhead factor to 1

Reply:
Also you may get conflict when you try to merge this as this wks file has changed. So first rebase.

Reply:
The size factor has been mentioned on the commit messages, you think its still good to add it there? Moreover, explicitly set it to 1?